wpilib/tests/test_solenoid.py

```diff
@@ -92,10 +103,12 @@ def test_solenoidbase_getAll(wpilib, hal_data):
         s['value'] = True
         
     assert solenoid.getAll() == 0xFF
+    assert wpilib.SolenoidBase.getAll(0) == 0xFF
```

Comment:
We should add a test for setAll()... but that means my hal changes need to be merged.

Reply:
My hal changes are merged, so someone should add a setAll test...

Reply:
where is setAll?

Reply:
They just introduced something at the HAL layer, so I had made an assumption that it would show up in the high level API too:

https://github.com/wpilibsuite/allwpilib/blob/6e4f66cc8b9fd96d82ca4119fc337bfc00331f09/hal/src/main/native/athena/Solenoid.cpp#L124

However, my assumption is incorrect and you're right. It does not appear to be there.